react/components/Link.tsx

```diff
@@ -156,6 +156,7 @@ const Link: React.FunctionComponent<Props> = ({
   const linkElementProps = {
     target,
     href: hrefWithoutIframePrefix,
+    rel,
```

Comment:
Remove from line 59. linkProps already has rel.

Reply:
Hi! So you're suggesting removing `rel` from line 59 and keeping it in line 159? I'll test it and I'll let you know

Reply:
If I remove it from line 159 while keeping it in line 59 seems to be working